install/linux/usr/share/odemis/sim/secom-flim-sim.odm.yaml

```diff
@@ -112,6 +112,10 @@ Light: {
     role: "tc-scanner",
 }
 
+"SPT Detector Live":  {
+    role: "tc-detector",
```

Comment:
That detector is only good/better for _live_ view (in spot mode). It's not synchronised with the mirror scanner, so it cannot be used for acquisition. That's why the idea is to have it with role `tc-detector-live`, and the `APD` should still have a role `tc-detector`. The GUI model  should also have two separate components, and the settings stream should support receiving (in argument) either both the tc-detector and tc-detector-live, or just the tc-detector (in which case it'll also be used for live move).

Reply:
Got it. Will fix it. 👍